Aquality.Selenium/src/Aquality.Selenium/Browsers/Browser.cs

```diff
@@ -70,28 +68,24 @@ public TimeSpan ImplicitWaitTimeout
         /// Default value: <see cref="ITimeoutConfiguration.PageLoad"/>.
         /// Ignored for Safari cause of https://github.com/SeleniumHQ/selenium-google-code-issue-archive/issues/687.
         /// </summary>
-        public TimeSpan PageLoadTimeout
+        /// <param name="timeout">Desired page load timeout.</param>
+        public void SetPageLoadTimeout(TimeSpan timeout)
         {
-            set
+            if (!configuration.BrowserProfile.BrowserName.Equals(BrowserName.Safari))
             {
-                if (!configuration.BrowserProfile.BrowserName.Equals(BrowserName.Safari))
-                {
-                    Driver.Manage().Timeouts().PageLoad = value;
-                    pageLoadTimeout = value;
-                }
+                Driver.Manage().Timeouts().PageLoad = timeout;
+                pageLoadTimeout = timeout;
             }
         }
 
         /// <summary>
         /// Sets Selenium WebDriver AsynchronousJavaScript timeout. 
         /// Default value: <see cref="ITimeoutConfiguration.Script"/>.
         /// </summary>
-        public TimeSpan ScriptTimeout
+        /// <param name="timeout">Desired AsynchronousJavaScript timeout.</param>
+        public void SetScriptTimeout(TimeSpan timeout)
```

Comment:
consider moving these properties to a separate "BrowserTimeouts" object as it is done in Selenium lib. @DmitryBogatko what do you think about it?